cmd/helm/root.go

```diff
@@ -134,31 +135,44 @@ func newRootCmd(actionConfig *action.Configuration, out io.Writer, args []string
 	flags.ParseErrorsWhitelist.UnknownFlags = true
 	flags.Parse(args)
 
+	commandGroups := templates.CommandGroups{
+		{
+			Message: "Release Commands:",
+			Commands: []*cobra.Command{
+				// release commands
+				newInstallCmd(actionConfig, out),
+				newListCmd(actionConfig, out),
+				newGetCmd(actionConfig, out),
+				newStatusCmd(actionConfig, out),
+				newUpgradeCmd(actionConfig, out),
+				newHistoryCmd(actionConfig, out),
+				newRollbackCmd(actionConfig, out),
+				newReleaseTestCmd(actionConfig, out),
+				newTemplateCmd(actionConfig, out),
```

Comment:
`helm template` might not be a good fit here, as it doesn't interact with the release ledger. Perhaps it should be placed under the other group.

Reply:
i agree too, @waveywaves what do you think? ,should me change back?

Reply:
update